src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -222,13 +273,15 @@ export class WorkspaceSavedObjectsClientWrapper {
       attributes: T,
       options?: SavedObjectsCreateOptions
     ) => {
-      if (isWorkspacesLikeAttributes(attributes)) {
-        await this.validateMultiWorkspacesPermissions(
-          attributes.workspaces,
-          wrapperOptions.request,
-          WorkspacePermissionMode.Management
-        );
+      const workspacePermitted = await this.validateWorkspaceAttributesPermitted(
```

Comment:
This validation will be failed if user does not pass workspaces attributes. I think that's a incorrect behavior.